catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/test/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformerTest.java

```diff
@@ -287,6 +287,108 @@ public void xmlAttributesCanBeMapped() {
     assertThat(getAttributeValue(Core.METACARD_OWNER, metacard), is("value3"));
   }
 
+  @Test
+  public void resourceSizeInBytes() {
```

Comment:
These tests are unrelated to the code I changed but the complexity coverage dropped slightly as a result of my change. I added these tests so I wouldn't have to lower the coverage limit.

Reply:
:true: :hero: